lobsterpy/featurize/core.py

```diff
@@ -208,11 +208,7 @@ def get_lobsterpy_cba_dict(path_to_lobster_calc: str, bonds: str) -> dict:
         icohplist_path = req_files_lobsterpy.get("icohplist_path")
 
         which_bonds = bonds
-
-        if which_bonds == "all":
-            bond_type = "all_bonds"
-        elif which_bonds == "cation-anion":
-            bond_type = "cation_anion_bonds"
+        bond_type = f"{which_bonds}_bonds"
```

Comment:
I personally think that this is the problem. Not the string.

Reply:
But in atomate2, we already have the snake case for this parameter. Can you please be specific what is the problem there?